HELP.md

```diff
@@ -26,11 +26,13 @@ This group contains general purposes helper tasks, potentially useful for a wide
 
 ### Properties
 
-| Name                               | Default Value | ENV Override                              | Description                                                                                                                                                              |
-| ---------------------------------- | ------------- | ----------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `RequiredPowerShellModules`        | @{}           | `ZF_REQUIRED_PS_MODULES`                  | A hashtable of PowerShell modules to install and import. The keys are the module names and the values are hashtables with the following properties: version, repository. |
-| `SkipEnsureGitHubCli`              | $true         | `ZF_SKIP_ENSURE_GITHUB_CLI`               | When true, ZeroFailed will skip the check for whether the GitHub CLI is installed. Default is true.                                                                      |
-| `SkipZeroFailedModuleVersionCheck` | $false        | `ZF_SKIP_ZEROFAILED_MODULE_VERSION_CHECK` | When true, ZeroFailed will skip the check for a newer version of the ZeroFailed module. Default is false.                                                                |
+| Name                               | Default Value | ENV Override                              | Description                                                                                                                                                                                                                                                                      |
+| ---------------------------------- | ------------- | ----------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `OnEnterActions`                   | @()           |                                           | A collection of scriptblocks that will be run as part of InvokeBuild's 'Enter-Build' functionality. Extensions and processes can register their own actions by calling `$script:OnEnterActions.Add($myScriptBlock)`.                                                             |
```

Comment:
Is it worth making it explicit that this these two are "Extensibility Points" i.e. "Extensibility Point: collection of scriptblocks...."

Reply:
I've push an update